hphp/runtime/vm/jit/irlower-conv.cpp

```diff
@@ -198,17 +198,23 @@ void cgConvDblToInt(IRLS& env, const IRInstruction* inst) {
   v << cvttsd2siq{src, d};
   v << cmpq{indef, d, sf};
 
+  // For ARM, use the cc which ignores unordered results for the first
+  // condition. Then use the ARM equivalent cc to check for the unordered
+  // result in the second condition.
+  ConditionCode cc_nb = (arch() == Arch::ARM) ? CC_GE : CC_NB;
+  ConditionCode cc_p = (arch() == Arch::ARM) ? CC_O : CC_P;
```

Comment:
Is it just coincidental that you can just use different CC's here?  That is, does `CC_GE` on ARM not actually do the same thing as `CC_NB` on x64, but it just so happens that we can still just do two checks and get the same semantics?